projects/1.20-fabric/assets/arknights-skadi-orca-plushie/skadi_orca_plushie/lang/zh_cn.json

```
@@ -0,0 +1,5 @@
+{
+  "block.skadi_orca_plushie.orca_plushie": "斯卡蒂的虎鲸抱枕",
+  "orca_plushie.tooltip": "非常柔软，摔落在上面可减免大部分摔落伤害",

```

```suggestion
  "orca_plushie.tooltip": "非常柔软，可吸收部分摔落伤害",
```

这句主要是忠于原文，顺带节省不必要的语素；有异议可以提

* 大部分→原文里没看到表示“大”的程度
* 吸收→原文用的abosorb，推测作者是想用类似缓冲材料的感觉，但是“吸收伤害”会给人一种回血的错觉；“减免”其实从功能性上没什么问题。这点改不改随你
* 删去条件→主要考虑到摔落伤害作为术语也不会引起歧义，如果为了防止“拿着会减伤”的歧义的话可以忽略我的修改

---

游戏里就是像蜂蜜块那种的缓冲垫